internal/config/docs.toml

```diff
@@ -0,0 +1,624 @@
+[WebServer.MFA]
+# The FQDN of where the Operator UI is served. When serving locally, the value should be `localhost`.
+RPID = 'localhost' # Example
+# The origin URL where WebAuthn requests initiate, including scheme and port. When serving locally, the value should be `http://localhost:6688/`.
+RPOrigin = 'http://localhost:6688/' # Example
+
+# The TLS settings apply only if you want to enable TLS security on your Chainlink node.
+[WebServer.TLS]
+# The location of the TLS certificate file.
+CertPath = '/home/$USER/.chainlink/tls/server.crt' # Example
+# The hostname configured for TLS to be used by the Chainlink node. This is useful if you configured a domain name specific for your Chainlink node.
+Host = 'tls-host' # Example
+# The location of the TLS private key file.
+KeyPath = '/home/$USER/.chainlink/tls/server.key' # Example
+# The port used for HTTPS connections. Set this to `0` to disable HTTPS. Disabling HTTPS also relieves Chainlink nodes of the requirement for a TLS certificate.
+HTTPSPort = 6689 # Default
+# Forces TLS redirect for unencrypted connections.
+ForceRedirect = false # Default
+
+[JobPipeline]
+# Defines the maximum size for HTTP requests and responses made by `http` and `bridge` adapters.
+HTTPRequestMaxSize = '32768' # Default
+# Defines the default timeout for HTTP requests made by `http` and `bridge` adapters.
+DefaultHTTPRequestTimeout = '15s' # Default
+# Enables the External Initiator feature. If disabled, `webhook` jobs can ONLY be initiated by a logged-in user. If enabled, `webhook` jobs can be initiated by a whitelisted external initiator.
+ExternalInitiatorsEnabled = false # Default
+# The maximum time allowed for a single job run. If it takes longer, it will exit early and be marked errored. If set to zero, disables the time limit completely.
+MaxRunDuration = '10m' # Default
+# In order to keep database size manageable, Chainlink nodes will run a reaper that deletes completed job runs older than a certain threshold age. `ReaperInterval` controls how often the job pipeline reaper will run.
+#
+# Set to `0` to disable the periodic reaper.
+ReaperInterval = '1h' # Default
+# Determines the age limit for job runs. Completed job runs older than this will be automatically purged from the database.
+ReaperThreshold = '24h' # Default
+# Some jobs write their results asynchronously for performance reasons such as OCR. `JOB_PIPELINE_RESULT_WRITE_QUEUE_DEPTH` controls how many writes will be buffered before subsequent writes are dropped.
+#
+# Do not change this setting unless you know what you are doing.
+ResultWriteQueueDepth = 100 # Default
+
+[FluxMonitor]
+DefaultTransactionQueueDepth = 1 # Default
+# Enable transaction simulation for Flux Monitor.
+SimulateTransactions = false # Default
+
+[OCR2]
+ContractConfirmations = 3 # Default
+BlockchainTimeout = '20s' # Default
+ContractPollInterval = '1m' # Default
+ContractSubscribeInterval = '2m' # Default
+ContractTransmitterTransmitTimeout = '10s' # Default
```

Comment:
Note that these correspond directly to internal OCR configuration variables. See: https://github.com/smartcontractkit/libocr/blob/master/offchainreporting2/oracle.go#L17

It probably makes sense to just link to the OCR2 docs for each of these.

I wonder if we could even be clever enough to pull the exact SHA of libocr we use for the generated docs and include a link to it?